res/values/strings.xml

```diff
@@ -183,133 +183,183 @@ there are any).</string>
-    <string name="np_desc_septa">Pennsylvania, Philadelphia</string>
-    <string name="np_desc_sydney">Australia, New South Wales, Sydney</string>
-    <string name="np_desc_met">Australia, Victoria, Melbourne</string>
-    <string name="np_desc_jet">Israel, Jerusalem</string>
-    <string name="np_desc_paca">Provence-Alpes-Côte d\'Azur</string>
-    <string name="np_desc_hsl">Helsinki</string>
-    <string name="np_desc_paris">Paris</string>
-    <string name="np_desc_nz">Wellington, Auckland</string>
-    <string name="np_desc_usny" tools:keep="@string/np_desc_usny">MTA, CDTA, Port Authority, Staten Island Ferry, NFTA, Rockland County, Rochester-Genesee Regional Transportation Authority</string>
-    <string name="np_desc_spain">Madrid buses (EMT), Barcelona (TMB), Basque (Moveuskadi)</string>
-    <string name="np_desc_mersey">Liverpool City Region</string>
-    <string name="np_desc_br">São Paulo, Rio de Janeiro, Belo Horizonte and Porto Alegre</string>
-    <string name="np_desc_br_floripa" translatable="false">SIM</string>
-    <string name="np_desc_vrs">Cologne, Bonn</string>
-    <string name="np_desc_frenchsouthwest">Poitou-Charentes, Limousin, Aquitaine, Midi-Pyrénées, Bordeaux, Toulouse</string>
-    <string name="np_desc_frenchsouthwest_networks" translatable="false">SNCF, TransGironde, TBC, Tisséo</string>
-    <string name="np_desc_francenortheast">Strasbourg, Metz, Lille, Nancy</string>
-    <string name="np_desc_francenortheast_networks" translatable="false">SNCF, CTS, Transpole, STAN, LE MET</string>
-    <string name="np_desc_francenorthwest">Bretagne, Pays de la Loire, Centre, Basse Normandie, Haute Normandie</string>
-    <string name="np_desc_francenorthwest_networks" translatable="false">LILA, Star, TAN, IRIGO, SNCF</string>
-    <string name="np_desc_it">Roma, Milano, Torino, Venezia, Palermo, Trento</string>
-    <string name="np_desc_it_networks" translatable="false">ATM, GTT, AMAT, ACTV</string>
-    <string name="np_desc_ontario">Ottawa, Toronto</string>
-    <string name="np_desc_quebec">Deux-Montagnes, Laval, L\'Assomption, Outaouais, Sud-Ouest, Quebec, Haut-Saint-Laurent, Lanaudière, La Presqu\'Île, Laurentides, Montreal, Les Moulins, Vallée du Richelieu, Chambly-Richelieu-Carignan, Roussillon, Sorel-Varennes, Le Richelain, Sherbrooke, Sainte-Julie</string>
 
-    <string name="np_name_db">Deutsche Bahn</string>
-    <string name="np_name_bayern">BAVARIA</string>
-    <string name="np_name_paris">PARIS</string>
-    <string name="np_name_sydney">SYDNEY</string>
-    <string name="np_name_stockholm">STOCKHOLM</string>
-    <string name="np_name_wien">VIENNA</string>
-    <string name="np_name_spain">Spain</string>
+    <string name="np_region_usa">United States of America</string>
+    <string name="np_name_rtachicago">RTA</string>
+    <string name="np_desc_rtachicago">Chicago</string>
+    <string name="np_desc_rtachicago_networks" translatable="false">cta, Metra, pace</string>
```

Comment:
I took that from the logos on the website http://rtachicago.com/.
However on closer examination they tend to write Pace and CTA. I think I will change it to this style.